### PR TITLE
Add a viewer for COSMO surfaces

### PR DESCRIPTION
**Pull request by y-kusaka**
I suppose I did rebase.

Add a viewer for COSMO surfaces

vis_poly/cosmo_poly.py
- read .cosmo or gout file to import cosmo data.
- output polygonized cosmo surfaces for visualization.
- used to_sigma.py for parsing.
- used open3d package for surface reconstruction (mesh generation)
- used pyvista package for visualization.

profiles/to_sigma.py
- clean: unnecessary 'print' at L158 removed.

### CODE REVIEW

File: vis_poly/cosmo_poly.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
+"""vizualise solvent accessible surface in .cosmo data"""
```
Comment (ianhbell) [inline vis_poly/cosmo_poly.py:1]: Can you add contact information at the top here?

File: vis_poly/cosmo_poly.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        self.pcd.points = o3d.utility.Vector3dVector(xyz)
+        self.pcd.colors = o3d.utility.Vector3dVector(intensity)
+        self.pcd.normals = o3d.utility.Vector3dVector(normals)
+        self.pcd.normalize_normals
+
+        # atom
+        self.pcd_atom.points = o3d.utility.Vector3dVector(xyz_atom)
+      
+        # a point cloud is converted to a surface mesh using Ball-Pivoting Algorithm.
+        # other possible surface reconstruction algoligthms: Octree, Poission, Powercrust, Delaunay including Alpha shapes, tight cocone etc.
```
[the remark this answers is not in the document]
Comment (y-kusaka) [reply vis_poly/cosmo_poly.py]: modified.

File: vis_poly/cosmo_poly.py (hunk 1 above)
Comment (y-kusaka) [reply vis_poly/cosmo_poly.py:1]: added.

**Comment by y-kusaka** [comment]
I did too many commits. sorry.  I should squash them before push. I'll handle this tomorrow.

File: vis_poly/README.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+- `.cosmo`  from Gaussian09
+- `.gout`  from GAMESS
+
+Samples can be found in profiles/DMol3_TEST, profiles/GAMESS_TEST, profiles/GAUSSIAN09_TEST and profiles/UD/cosmo.
+
+### output filetypes (`outpath`):
+
+In the `outpath`  argument, a output file with one of the following extensions must be given. The script supports general mesh formats  
+
+- `ply`: stanford polygon
```
Comment (ianhbell) [inline vis_poly/README.md:40]: Stanford

File: vis_poly/README.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
+# cosmo_poly.py
+
+This `cosmo_poly.py` script visualizes cosmo surface from cosmo segment data.
+
+![cosmo surface image](ethanol.png)
+
+## usage
+
+In a termnial at `vis-poly` folder,
```
Comment (ianhbell) [inline vis_poly/README.md:9]: terminal

File: vis_poly/README.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+```termnial
```
Comment (ianhbell) [inline vis_poly/README.md:10]: terminal

**Comment by ianhbell** [comment]
It's ok, I can squash-merge at the end, which is usually how I handle pull requests.  That avoids accidents persisting in the main codebase.

File: vis_poly/README.md (hunk 3 above)
Comment (y-kusaka) [reply vis_poly/README.md]: modified.

File: vis_poly/README.md (hunk 4 above)
Comment (y-kusaka) [reply vis_poly/README.md]: modified.

File: vis_poly/README.md (hunk 5 above)
Comment (y-kusaka) [reply vis_poly/README.md:10]: modified.

**Comment by ianhbell** [comment]
Are we all set here?

**Comment by y-kusaka** [comment]
> Are we all set here?

I guess so.

**Comment by ianhbell** [comment]
Ok good, I'll squash+merge tomorrow.

**Comment by ianhbell** [comment]
Very nice work!

**Pull request merged**